.github/workflows/fips-provider.yml

```diff
@@ -92,6 +92,9 @@ jobs:
       - name: copy the provider
         run: |
           cp -a build/providers/fips.so build-3.0/providers/fips.so
+      - name: removing incompatible testcase
```

Comment:
This is not a good way to omit a test.  Do it with a FIPSversion conditional in the test data.

Reply:
So should I do a PR against 3.0 branch?

Reply:
Alternative PR (against 3.0 branch) in #19904